baseline.yaml

```diff
@@ -649,6 +649,30 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+- id: OSPS-DO-14
+    maturity_level: 3
+    category: Documentation
+    criteria: |
+       The project documentation MUST provide a 
+       descriptive statement when releases or 
+       versions are no longer supported and that 
+       will no longer receive security updates.  
+       
+       This should be provided both in human and 
+       machine-readable formats. 
```

Comment:
```suggestion
```

I like machine-processable formats, but I don't know of one for this use. I certainly don't know of any with widespread acceptance. Security-insights.yml can say when the project ends, but I don't think it can do more.

If there is one, please identify that (at least as an option) in the "implementation" section.